src/frontend/src/btc/components/convert/BtcConvertTokenWizard.svelte

```diff
@@ -68,6 +69,14 @@
 	// Docs: https://github.com/dfinity/ic/blob/master/rs/bitcoin/ckbtc/minter/README.adoc#bitcoin-to-ckbtc
 	$: destinationAddress = $btcAddressStore?.[$destinationToken.id]?.data;
 
+	onMount(() => {
+		loadBtcPendingSentTransactions({
```

Comment:
Is it correct to do it that way UX wise, shouldn't the information be loaded before the modal open under the "Hold tight..." backdrop loader?

Reply:
If for some reason the call would actually takes time - think low end device or slow(er) internet connection - would the UI shift and would the UX be impacted by the data being not yet loaded?

Reply:
I see, well not sure. On one hand, as you say, if it's fast, it's fast, on the other, a disabled button without any hint of the why is not necessarely user friendly. Don't know what's best. I approve the PR and let you decide ;)

Reply:
That said, can you add a test that validates the button is disabled until loaded?

Reply:
Tests have been added.

I would say, for the very first version, it should be acceptable. I will keep an eye on it while testing and if I notice something, I'll come back to this topic.

Reply:
Alright. Thanks for the test